src/main/java/com/thealgorithms/datastructures/hashmap/hashing/ItineraryTickets.java

```diff
@@ -0,0 +1,29 @@
+package com.thealgorithms.datastructures.hashmap.hashing;
+
+import java.util.HashMap;
+
+/**
+ * This class represents an itinerary of tickets. Given a list of tickets, it finds the
+ * complete itinerary.
+ */
+public class ItineraryTickets {
+
+    /**
+     * This method finds the starting point of the itinerary.
+     *
+     * @param tickets A map of tickets where the key is the departure point, and the value is the destination.
+     * @return The starting point of the itinerary.
+     */
+    public static String getStart(HashMap<String, String> tickets) {
```

Comment:
`tickets` are not modified here - they can be marked as `final`.
```suggestion
    public static String getStart(final HashMap<String, String> tickets) {
```